opensrp-reveal/src/main/java/org/smartregister/reveal/view/ListTasksActivity.java

```diff
@@ -693,6 +705,22 @@ public boolean isMyLocationComponentActive() {
         return revealMapHelper.isMyLocationComponentActive(this, myLocationButton);
     }
 
+    @Override
+    public void displayMarkStructureInactiveDialog() {
+        AlertDialogUtils.displayNotificationWithCallback(this, R.string.mark_location_inactive,
+                R.string.confirm_mark_location_inactive, R.string.confirm, R.string.cancel, new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        switch (which) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/app/OpenSRP/opensrp-client-reveal/pullRequest?prid=4628033)